appserver/tests/payara-samples/samples/concurrency/pom.xml

```diff
@@ -94,7 +94,7 @@
         <dependency>
             <groupId>jakarta.platform</groupId>
             <artifactId>jakarta.jakartaee-api</artifactId>
-            <version>10.0.0-RC1</version>
+            <version>${jakartaee-api.version}</version>
```

Comment:
Is this required? It's already defined in dependency management in payara-bom

Reply:
yes it is not needed. Now removed